test/e2e/cloud/gcp/cluster_upgrade.go

```diff
@@ -95,7 +95,7 @@ func masterUpgrade(f *framework.Framework, v string) error {
 	case "gce":
 		return masterUpgradeGCE(v, false)
 	case "gke":
-		return framework.MasterUpgradeGKE(f.Namespace.Name, v)
+		return framework.MasterControlPlaneUpgradeGKE(f.Namespace.Name, v)
```

Comment:
Need to remove `Master` from the function name.

Reply:
Yes, I will change it immediately. Thank you.

Reply:
done.